### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rishabh510**
- **What does this PR do?**

1. Better highlighted text headings.
2. Removed '' from API table default value.

- **Any background context you want to provide?**

- **Screenshots and/or Live Demo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default class App extends Component {
     <td>value</td>
     <td>string / number</td>
     <td><strong>true</strong></td>
-    <td>''</td>
+    <td></td>
```
Comment (akashnimare) [inline README.md]: @apollonian is this good?

**Comment by akashnimare** [comment]
@Rishabh510 can you fix the merge conflicts?

File: README.md (hunk 1 above)
Comment (apollonian) [reply README.md]: Yep! separator is not an empty string by default so this change looks good.

Comment (akashnimare) [reply README.md]: Cool then.

Comment (akashnimare) [reply README.md]: @Rishabh510 can you keep it same?

Comment (Rishabh510) [reply README.md]: Sure

**Comment by ganga549** [comment]
I am getting the error, Can't resolve react in react-otp-input/lib
ERROR in ./node_modules/react-otp-input/lib/index.js
Module not found: Error: Can't resolve 'react' in '/Users/gangapd/learningnodejs/klogin/preact/node_modules/react-otp-input/lib'
 @ ./node_modules/react-otp-input/lib/index.js 13:13-29
 @ ./src/verify.jsx

**Pull request merged**